Gemfile

```diff
@@ -1,49 +1,51 @@
-source "https://rubygems.org"
+# frozen_string_literal: true
+
+source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby "3.0.2"
+ruby '3.0.2'
```

Comment:
I generally like to minimize the number of diffs when we're introducing rubocop to a repo at first.  If we want to go with single quotes, we can create a PR afterwards that introduces that single rule change so that reviewing the diff is very simple.